test/testmatrix.cpp

```diff
@@ -537,11 +537,40 @@ TEST(Matrix, luDecompPivoting)
     x = A.solve(rhs);
 
     CHECK_EQUAL((4*c - 3*b)/(a*c), x(0));
-    CHECK_EQUAL(3*(b - c)/c, x(1));
+    CHECK_EQUAL(-(3*(-b + c)/c), x(1));
```

Comment:
yes somehow he didn't simplify it far enough when in the end the other sign is added?